src/pnpq/devices/odl_ozoptics_650ml.py

```diff
@@ -47,8 +48,14 @@ def set_step(self, value):
         response = self.serial_command(cmd)
         return response
 
-    def current_step():
-        pass
+    def get_step(self) -> int:
+        response = self.serial_command("S?")
+        if "UNKNOWN" in response:
+            raise OdlGetPosNotCompleted(
+                f"Unknown position for ODL({self}): run find_home() first and then change or get the position"
+            )
+        step = response.split("Done")[0].split(":")[1]
```

Comment:
This is existing functionality, and I don't want to change it in this pull request.

However, this is a very strange way of parsing text, especially multi-line text. It's especially strange now that I can see what the text looks like in https://github.com/moonshot-nagayama-pj/PnPQ/issues/39

There is probably a better way to write this that is easier to understand. But we will have to examine that later.